File: Programming-languages-and-laboratory/Python/Exercises/30_compito_java_20190617/Pallacanestro.py
from Squadra import Squadra


class Pallacanestro(Squadra):
    def __init__(self, codice, sport, nome, vinte, perse, falli, punti_totali):
        super().__init__(codice, sport, nome, vinte, perse)
        self._falli = falli
        self._punti_totali = punti_totali

    def __str__(self):
        return super().__str__() + "\t" + str(self._punti_totali) + "\t" + str(self._falli) + "\t-\t" + self._sport

File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Java/Exercises/30_compito_java_2019_06_17/Pallacanestro.java
public class Pallacanestro extends Squadra {
	private double falli;
	private int puntiSegnati;
	private int numeroGiocatori;

	public Pallacanestro(int codice, String sport, String nomeSquadra, int vinte, int perse, double falli, int puntiSegnati) {
		super(codice, sport, nomeSquadra, vinte, perse);
		this.falli = falli;
		this.puntiSegnati = puntiSegnati;
		numeroGiocatori = 0;
	}

	public String toString() {
		return nomeSquadra + "\t" + codice + "\t" + vinte + "\t" + perse + "\t" + puntiSegnati + "\t" + falli + "\t-\t" + sport;
	}
	
	public void incrementaGiocatori(Pallacanestro p) {
		p.numeroGiocatori++;
	}
}